src/fish/README.md

```diff
@@ -17,10 +17,12 @@ Installs fish shell and Fisher plugin manager (optionally)
 
 | Options Id | Description | Type | Default Value |
 |-----|-----|-----|-----|
-| fisher | Install Fisher plugin manager | boolean | true |
-
-
+| `fisher` | Install [Fisher][fisher] plugin manager | boolean | `true` |
+| `fishtape` | Install [Fishtape][fishtape], 100% pure-Fish test runner | boolean | `false` |
 
 ---
 
 _Note: This file was auto-generated from the [devcontainer-feature.json](https://github.com/meaningful-ooo/devcontainer-features/blob/main/src/fish/devcontainer-feature.json).  Add additional notes to a `NOTES.md`._
+
+[fisher]: https://github.com/jorgebucaran/fisher
+[fishtape]: https://github.com/jorgebucaran/fishtape
```

Comment:
Do not edit `README.md` directly as it is automatically generated from `devcontainer-feature.json` and `NOTES.md`.